src/layouts/shared/page-header.tsx

```diff
@@ -1,10 +1,14 @@
 import { CommonText, PageText, useTranslation } from '@atb/translations';
 import Link from 'next/link';
 import style from './page-header.module.css';
-import { MonoIcon } from '@atb/components/icon';
+import { useDarkMode } from '@atb/modules/theme';
+import Image from 'next/image';
+import { getOrgData } from '@atb/modules/org-data';
 
 export default function PageHeader() {
   const { t } = useTranslation();
+  const [isDarkMode] = useDarkMode();
```

Comment:
Not sure why `isDarkMode` is wrapped in an array, but it seems to be the pattern used elsewhere.